public/src/app/components/success/success.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { AppService } from 'src/app/app.service';
+
+@Component({
+  selector: 'app-success',
+  templateUrl: './success.component.html',
+  styleUrls: ['./success.component.scss']
+})
+export class SuccessComponent implements OnInit {
+  public scanId: string;
+  public processFinished: boolean;
+
+  constructor(
+    private readonly router: Router,
+    private readonly service: AppService
+  ) {}
+
+  ngOnInit(): void {
```

Comment:
```suggestion
  public ngOnInit(): void {
```

https://github.com/NeuraLegion/nexploit-cli/pull/100#discussion_r494100700